packages/eas-cli/src/credentials/ios/api/GraphqlClient.ts

```diff
@@ -0,0 +1,233 @@
+import nullthrows from 'nullthrows';
+
+import { AppleAppIdentifierMutation } from '../../../graphql/mutations/credentials/AppleAppIdentifierMutation';
+import { AppleDistributionCertificateMutation } from '../../../graphql/mutations/credentials/AppleDistributionCertificateMutation';
+import { AppleProvisioningProfileMutation } from '../../../graphql/mutations/credentials/AppleProvisioningProfileMutation';
+import { AppleTeamMutation } from '../../../graphql/mutations/credentials/AppleTeamMutation';
+import { IosAppBuildCredentialsMutation } from '../../../graphql/mutations/credentials/IosAppBuildCredentialsMutation';
+import { IosAppCredentialsMutation } from '../../../graphql/mutations/credentials/IosAppCredentialsMutation';
+import { AppQuery } from '../../../graphql/queries/AppQuery';
+import { AppleAppIdentifierQuery } from '../../../graphql/queries/credentials/AppleAppIdentifierQuery';
+import { AppleDeviceQuery } from '../../../graphql/queries/credentials/AppleDeviceQuery';
+import { AppleDistributionCertificateQuery } from '../../../graphql/queries/credentials/AppleDistributionCertificateQuery';
+import { AppleProvisioningProfileQuery } from '../../../graphql/queries/credentials/AppleProvisioningProfileQuery';
+import { AppleTeamQuery } from '../../../graphql/queries/credentials/AppleTeamQuery';
+import { IosAppCredentialsQuery } from '../../../graphql/queries/credentials/IosAppCredentialsQuery';
+import { App } from '../../../graphql/types/App';
+import { AppleAppIdentifier } from '../../../graphql/types/credentials/AppleAppIdentifier';
+import { AppleDevice } from '../../../graphql/types/credentials/AppleDevice';
+import { AppleDistributionCertificate } from '../../../graphql/types/credentials/AppleDistributionCertificate';
+import { AppleProvisioningProfile } from '../../../graphql/types/credentials/AppleProvisioningProfile';
+import { AppleTeam } from '../../../graphql/types/credentials/AppleTeam';
+import {
+  IosAppBuildCredentials,
+  IosDistributionType,
+} from '../../../graphql/types/credentials/IosAppBuildCredentials';
+import { IosAppCredentials } from '../../../graphql/types/credentials/IosAppCredentials';
+import { Account } from '../../../user/Account';
+import { DistributionCertificate } from '../appstore/Credentials.types';
+
+export interface AppLookupParams {
+  account: Account;
+  projectName: string;
+  bundleIdentifier: string;
+}
+
+export default class IosGraphqlClient {
+  public async getAppAsync(appLookupParams: AppLookupParams): Promise<App> {
```

Comment:
```suggestion
export async function getAppAsync(appLookupParams: AppLookupParams): Promise<App> {
```
Please consider if `IosGraphqlClient` needs to be a class or if a module with functions would be sufficient. I see it currently holds no internal state (and it might be a good idea to keep it that way) so maybe it doesn't need to be a class?